### ISSUE TITLE
"Sequence contains more than one matching element." when switching from DebugView to C#

### ISSUE THREAD

**Issue by zspitz** [zspitz/ExpressionTreeVisualizer#115]
**Describe the bug**  
When switching from DebugView to C#, the visualizer fails to load with "Sequence contains more than one matching element."

**Version info:**

- Visual Studio version: 2019
- Visualizer location: Documents subfolder
- Version: 
- ExpressionTreeToString version: 3.2 (not yet committed)

**Additional context**  
Test object name: FactoryMethods.DynamicBinaryOperation  
Factory methods:  
```csharp
// using static System.Linq.Expressions.Expression
Dynamic(
    #CSharpBinaryOperationBinder,
    typeof(double), x, y
)
```

**Comment by zspitz** [comment]
I've seen this intermittently while debugging ExpressionTreeToString.

**Comment by zspitz** [comment]
Fixed 477fca3087f48792b4c3e9d41f20e6affc6fe80c
